cluster-install/install.sh

```diff
@@ -819,12 +820,11 @@ tool_is_installed() {
     local -r tool="$1"
     if [ -z "$tool" ]; then fatal "no tool name provided"; fi
 
-    if
-        command -v "$tool" &>/dev/null
+    if command -v "$tool" &>/dev/null; then
         return 0
-    then
-        return 1
     fi
+
+    return 1
```

Comment:
I must've been typing with my monitor off when I put this in. The `then` was in the complete wrong place and there was an extra newline for whatever reason. Overall, it made the tool install check mostly useless, which this change should fix (credit to Keir for noticing and testing the fix).